docs/index.md

```diff
@@ -1,639 +1,128 @@
-    manager for something that works universally across all languages!
+[Node :simple-nodedotjs:](https://flox.dev/blog/using-flox-to-create-portable-reproducible-nodejs-environments/){ .md-button }
+[Go :fontawesome-brands-golang:](https://flox.dev/blog/using-flox-to-create-portable-reproducible-go-environments/){ .md-button }
+[Python :fontawesome-brands-python:](https://flox.dev/blog/using-flox-to-create-portable-reproducible-python-environments/){ .md-button }
+[Rust :fontawesome-brands-rust:](https://flox.dev/blog/a-real-world-rust-project-with-flox/){ .md-button }
+[Ruby :material-language-ruby:](https://flox.dev/blog/making-ruby-projects-easier-to-share/){ .md-button }
 
-=== "Node.js"
+---
 
-    ```
-    $ cd my_project
+## Why Flox?
 
-    my_project $ flox init
-    Flox detected a package.json
+We encounter the same challenges, no matter the stack: inconsistent environments, dependency drift, and brittle build processes that don’t scale well across machines, teams, or deployment targets. Current solutions often add complexity and fragmentation.
 
-    Flox can add the following to your environment:
-    * nodejs 20.17.0 with npm bundled
-    * An npm installation hook
+Flox takes a different approach: it provides a consistent, language-agnostic workflow for managing environments, from local development to CI to production.
 
-    ! Would you like Flox to apply this suggestion?
-    You can always change the environment's manifest with 'flox edit'
-    > Yes
-      No
-      Show suggested modifications
-    [Use '--auto-setup' to apply Flox recommendations in the future.]
-    ...select yes...
-    ✨ Created environment 'my_project' (aarch64-darwin)
-    ✅ 'nodejs' installed to environment 'my_project'
+Use Flox to solve three common use cases: 
 
-    Next:
-      $ flox search <package>    <- Search for a package
-      $ flox install <package>   <- Install a package into an environment
-      $ flox activate            <- Enter the environment
-      $ flox edit                <- Add environment variables and shell hooks
-    ```
+- Reproducible dev environments
+- Reliable package management across systems 
+- Consistent builds from local to production
 
-    If your project contains a `package.json` or a `yarn.lock`,
-    you'll be asked whether you'd like Flox to install `npm` or `yarn`
-    with versions matching those in the `package.json` or `yarn.lock` file
-    if found in the Flox Catalog.
+### __Reproducible dev environments__
 
-    If you accept, it will also add a hook that calls either `npm install` or
-    `yarn` when activating the environment so that your dependencies are up to
-    date.
+Set up a [local developer environment](https://flox.dev/docs/tutorials/creating-environments/) that will work the same across multiple system types and architectures. Seamlessly switch between development environments across multiple language ecosystems using a consistent, unified workflow.
```

Comment:
```suggestion
Set up a [local developer environment](https://flox.dev/docs/tutorials/creating-environments/) that will work the same across multiple operating systems and architectures. Seamlessly switch between development environments across multiple language ecosystems using a consistent, unified workflow.
```

`system types` would include both operating system and architectures in my mind, so if we've split out architectures, it makes sense to split out operating systems as well.